src/components/Flex/Flex.tsx

```diff
@@ -47,7 +48,8 @@ const Flex: VibeComponent<FlexProps> & {
       style,
       ariaLabelledby,
       ariaLabel,
-      tabIndex
+      tabIndex,
+      "data-testid": dataTestId = getTestId(ComponentDefaultTestId.FLEX, id)
```

Comment:
I don't think this is a relevant change for this PR and in general, I don't think this component should have a test id since it's not functional, only UI

Reply:
This change added to the pr because I needed to click on the container which was a flex component. I think layout components can use data-testsid even if they are pure UI.
Also, our current guidelines are that every Vibe component should have data test id

Reply:
so I'm fine with allowing to pass a data-testid, but I don't think there should be a default one